frontend/src/__tests__/cypress/cypress/tests/mocked/modelCatalog/modelCatalog.cy.ts

```diff
@@ -161,12 +204,12 @@ describe('Model Catalog loading states', () => {
           namespace: 'opendatahub',
         },
         data: {
-          modelCatalogSources: '',
+          modelCatalogSources: JSON.stringify({ sources: [] }),
         },
       },
     );
     modelCatalog.visit();
-    modelCatalog.findModelCatalogEmptyState().should('exist');
+    cy.contains('Request access to model catalog').should('exist');
```

Comment:
This also should still use `modelCatalog.findModelCatalogEmptyState()`, no need to hard-code this string again